openfe/protocols/openmm_afe/equil_afe_settings.py

```diff
@@ -51,62 +56,126 @@ class LambdaSettings(SettingsBaseModel):
-
     """
+
+    # fmt: off
     lambda_elec: list[float] = [
         0.0, 0.25, 0.5, 0.75, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0,
         1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0,
     ]
+    # fmt: on
     """
     List of floats of lambda values for the electrostatics. 
-    Zero means state A and 1 means state B.
-    Length of this list needs to match length of lambda_vdw and lambda_restraints.
+    Zero means fully interacting (state A),
+    and one means annihilated (state B).
+    Length of this list needs to match length of lambda_vdw and
+    lambda_restraints.
     """
+    # fmt: off
     lambda_vdw: list[float] = [
         0.0, 0.0, 0.0, 0.0, 0.0, 0.05, 0.1, 0.2, 0.3, 0.4,
         0.5, 0.6, 0.65, 0.7, 0.75, 0.8, 0.85, 0.9, 0.95, 1.0,
     ]
+    # fmt: on
     """
     List of floats of lambda values for the van der Waals.
-    Zero means state A and 1 means state B.
-    Length of this list needs to match length of lambda_elec and lambda_restraints.
+    Zero means full interacting (state A) and one means decoupled (state B).
+    Length of this list needs to match length of lambda_elec and
+    lambda_restraints.
     """
+    # fmt: off
     lambda_restraints: list[float] = [
         0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
         0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
     ]
+    # fmt: on
     """
     List of floats of lambda values for the restraints.
-    Zero means state A and 1 means state B.
-    Length of this list needs to match length of lambda_vdw and lambda_elec.
+    Zero means no restraints are applied (state A), and
+    one means restraints are fully applied (state B).
+
+    Note: The length of this list needs to match length of lambda_vdw and lambda_elec.
     """
 
-    @field_validator('lambda_elec', 'lambda_vdw', 'lambda_restraints')
+    @field_validator("lambda_elec", "lambda_vdw", "lambda_restraints")
     def must_be_between_0_and_1(cls, v):
         for window in v:
             if not 0 <= window <= 1:
-                errmsg = ("Lambda windows must be between 0 and 1, got a"
-                          f" window with value {window}.")
+                errmsg = (
+                    "Lambda windows must be between 0 and 1, got a"
+                    f" window with value {window}."
+                )
                 raise ValueError(errmsg)
         return v
 
-    @field_validator('lambda_elec', 'lambda_vdw', 'lambda_restraints')
+    @field_validator("lambda_elec", "lambda_vdw", "lambda_restraints")
     def must_be_monotonic(cls, v):
 
         difference = np.diff(v)
+        monotonic = np.all(difference >= 0)
 
-        if not all(i >= 0. for i in difference):
-            errmsg = f"The lambda schedule is not monotonic, got schedule {v}."
+        if not monotonic:
+            errmsg = (
+                "The lambda schedule is not monotonically increasing, "
+                f"got the follow schedule {v}."
```

Comment:
Maybe something like this?
```suggestion
                f"got the following schedule: {v}."
```